action.yml

```diff
@@ -83,9 +90,20 @@ runs:
           runs/wokwi/reports/metrics.csv
           runs/wokwi/reports/synthesis/1-synthesis*
 
-    - name: Create PNG
+    # Create and store PNG...
+    - name: Render SVG from GDS
+      shell: bash
+      run: ./tt/tt_tool.py --create-svg
+
+    - name: Convert SVG to PNG
       shell: bash
-      run: ./tt/tt_tool.py --create-png
+      run: rsvg-convert --unlimited gds_render.svg -s <(echo 'text{display:none;}') -o gds_render_png24.png --no-keep-image-data
```

Comment:
What's the purpose of `<(echo 'text{display:none;}')` ?

Reply:
That hides text labels when rendering the PNG (which otherwise become a cluttered mess). `rsvg-convert`'s `-s` parameter expects a CSS file as input; this inlines it so a separate file isn't needed.

Reply:
Thanks for explaining! Can you please add a comment to the yaml that explain this?

Reply:
OK, comment added in https://github.com/TinyTapeout/tt-gds-action/pull/1/commits/bae56eec67d05458b812492680366a35f5617cf3